docker-compose.yml

```diff
@@ -254,7 +254,7 @@ services:
       IDP_NAME: "IdP3"
 
   ssp-sp1.local:
-    build: .
+    image: silintl/ssp-base:9.3.0
```

Comment:
Hmm... this concerns me slightly, though it is a nice simple fix for the problem.

I supposed since this is just to be "some website that supports SAML", for testing the actual IDP and Hub behavior (which _are_ using the latest code), it's probably totally fine. It just _seems_ undesirable, at least at first.

Okay, no objection. Just needed to think through that. 🙂

Reply:
Right, I even have intent to use something even simpler than ssp-base for this. Either someone else's SimpleSAMLphp docker image, or one I create just as a plain-and-simple SP implementation.